sdk/runtime/go.sum

```diff
@@ -1,12 +1,77 @@
+github.com/alecthomas/template v0.0.0-20160405071501-a0175ee3bccc/go.mod h1:LOuyumcjzFXgccqObfd/Ljyb9UuFJ6TxHnclSeseNhc=
```

Comment:
Ooooof, that's a lot of transitive dependencies. I thought prometheus would be lighter. Just to make sure we're not leaking anything unnecessary here, could you run `go mod tidy` once more?

Reply:
I ran it about 3 times already.